include/cling/Utils/Paths.h

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
```

Comment:
I am confused. Who wrote this test? You our a colleague with that name?
I am fine with keeping that name there, as long as you can guarantee the person is aware that this piece of code is donated to the cling project.